pysat/instruments/templates/template_instrument.py

```diff
@@ -222,21 +219,34 @@ def list_files(tag='', inst_id='', data_path=None, format_str=None):
     ----
     The returned Series should not have any duplicate datetimes. If there are
     multiple versions of a file the most recent version should be kept and the
-    rest discarded. This routine uses the pysat.Files.from_os constructor, thus
-    the returned files are up to pysat specifications.
+    rest discarded. This routine uses the `pysat.Files.from_os` constructor,
+    thus the returned files are up to pysat specifications.
 
     Multiple data levels may be supported via the 'tag' input string.
-    Multiple instruments via the inst_id string.
+    Multiple instruments via the `inst_id` string.
 
     """
 
     if format_str is None:
         # User did not supply an alternative format template string
-        format_str = 'example_name_{year:04d}_{month:02d}_{day:02d}.nc'
+        format_str = 'example-name_{year:04d}_{month:02d}_{day:02d}.nc'
 
     # We use a pysat provided function to grab list of files from the
-    # local file system that match the format defined above
-    file_list = pysat.Files.from_os(data_path=data_path, format_str=format_str)
+    # local file system that match the format defined above. This example
+    # is set to parse filenames from the local system using the delimiter '_'.
+    # Alternately, leaving the `delimiter` to the default of `None` will
+    # engage the fixed width filename parser. If there is not a common
+    # delimiter then the fixed width parser is suggested though not always
+    # required. Given the range of standards compliance across the decades of
```

Comment:
```suggestion
    # engage the fixed width filename parser. Given the range of standards
    # compliance across the decades of
```

And fix spacing.  Recommend removing that sentence because from this level, you can't specify which parser is used.  I was reading this today and it took me a moment before I realized I had no control over that aspect.

Reply:
A developer can control which parser is used when starting here as a template. If delimiter=None or left unassigned then the fixed width parser is used.

 I don't understand the fix spacing comment.

Reply:
"If there is not a common delimiter then the fixed width parser is suggested though not always required" is not accurate in this setting because you can ONLY differentiate between the two by providing or not providing a delimiter. 

"fix spacing" means that I don't know if the lines are now too long or short as a part of the whole comment block.

Reply:
It is correct that you can only provide a single given delimiter or use the fixed width parser. In practice though, the delimited parser will likely work on filenames that don't have even a single instance of the given delimiter. I think I even test for that.  

> "fix spacing" means that I don't know if the lines are now too long or short as a part of the whole comment block.

Got it!

Reply:
GitHub shows me that this hasn't been addressed?  Which is confusing because this is marked as outdated.

Reply:
Sometimes I think it gets confused. I updated the comment a bit to specifically note I am talking about modifying the function call `from_os` below the comment.